docs/conf.py

```diff
@@ -41,7 +41,7 @@
 
 # General information about the project.
 project = 'Rubicon'
-copyright = '2018, Russell Keith-Magee'
+copyright = '2014, Russell Keith-Magee'
```

Comment:
So AIUI, no. [There's some argument about whether copyright dates are needed at all](http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/), but when they are used, the date is [the earliest date of copyright assertion](https://stackoverflow.com/a/2391555).

Reply:
Right - makes sense that the earliest copyright year should be recorded. I suppose you could use a range of years instead (2014-2020) to record both the earliest and latest years. Then again, as pointed out in [this answer on SO](https://stackoverflow.com/a/20911485), the year isn't all that relevant for individual copyrights anyway, as in that case the copyright length is based on the death of the author, not when the work was created.

> There is no reason at all for an individual to update the copyright year, because in the U.S. and Europe the life of copyright is the life of the author plus 70 years (50 years in some other countries like Canada and Australia). Extending the date does not extend the copyright. This also applies when a page has multiple contributors none of which are corporations.

Honestly I'm fine with just listing the earliest year if it's sufficient in this case - that way we don't have to update the copyright notice every year.